package.json

```diff
@@ -33,6 +33,8 @@
     "opn": "^1.0.0",
     "serve-index": "^1.1.4",
     "serve-static": "^1.4.0",
-    "wiredep": "^2.0.0"
+    "wiredep": "^2.0.0",
+    "gulp-mocha": "~2.0.0",
```

Comment:
https://github.com/ccraciun/jsDip/blob/master/test/bower.json#L6 already has mocha and chai -- but we need a frontend package manager like browserify to easily include them from the test js file.